### ISSUE TITLE
Fullscreen Button Hides Toolbar Instead of Maximizing to Fullscreen

### ISSUE THREAD

**Issue by Devmoni** [llaske/sugarizer#1689]
### Description:
When the fullscreen button is clicked, it hides the toolbar. The expected behavior is for the app to enter fullscreen mode (using the entire device screen), but the toolbar should remain visible. The fullscreen button should be replaced by the unfullscreen button.

### Expected Behavior:
- The fullscreen button should maximize the app to fullscreen, keeping the toolbar visible.
- The fullscreen button should be replaced by the unfullscreen button.
- When the unfullscreen button is clicked, the app should exit fullscreen and the fullscreen button should reappear.
### Current Behavior:
- The fullscreen button hides the toolbar, which is not the expected behavior.

**Comment by Devmoni** [comment]
As I have just implemented this functionality for a single activity to test the result, I think it needs to be implemented separately for each activity to ensure consistency across the app.
@llaske

**Comment by llaske** [comment]
No. The expected behavior is to hide the toolbar and use all the space need.
It's different from the full screen mode of the browser.
Most of Sugarizer users are using a tablet where the app use already the whole screen. So implementing your full screen mode has no sense for them.
